ui/js/components/auth.js

```diff
@@ -117,70 +121,24 @@ treeherder.component("login", {
             };
 
             ctrl.setLoggedIn = function (newUser) {
+                const userSession = localStorageService.get('userSession');
+
                 newUser.loggedin = true;
-                localStorageService.set("user", newUser);
+                newUser.fullName = userSession.fullName;
+
                 ctrl.user = newUser;
                 ctrl.onUserChange({ $event: { user: newUser } });
 
+                // start session renewal process
+                if (userSession && userSession.renewAfter) {
+                    authService.resetRenewalTimer(userSession);
+                }
             };
 
             ctrl.setLoggedOut = function () {
-                localStorageService.set("user", loggedOutUser);
-                localStorageService.set('taskcluster.credentials', {});
+                authService.logout();
```

Comment:
I assume the issue is this call to authService.logout, since the other one logs to the console and I don't see any console output.  Adding some breakpoints shows that this is being called from the storage event listener.  Although setting a breakpoint on the event listener doesn't trigger, so maybe that's the debugger being confused by webpack, I don't know.  I'd suggest adding some console.logs in here to see how this is working..